hypothesis-python/src/hypothesis/extra/pytestplugin.py

```diff
@@ -120,42 +124,36 @@ def pytest_runtest_makereport(item, call):
             'Hypothesis',
             '\n'.join(item.hypothesis_report_information)
         ))
+    if hasattr(item, 'hypothesis_statistics') and report.when == 'teardown':
+        # Running on pytest <= 3.5 where user_properties doesn't exist, fall
```

Comment:
This says `<= 3.5` but the code before checks for `< 3.5`. Which is it?

Reply:
The code is correct, and the comment now fixed.